lua/ttt2/libraries/fonts.lua

```diff
@@ -0,0 +1,82 @@
+---
+-- font library functions
+-- adds support for advanced fonts (fonts with mipmapping)
+-- @author Mineotopia, LeBroomer
+
+AddCSLuaFile()
+
+if SERVER then return end
+
+local surface = surface
+local mathMax = math.max
+local isvector = isvector
+local tonumber = tonumber
+local tostring = tostring
+
+fonts = {}
+fonts.fonts = {}
+fonts.scales = {1, 1.5, 2, 2.5}
+
+---
+-- Gets the scale modifer based on a given scale.
```

Comment:
This should be more expressive, what is a scale modifier based on a given scale? What does this function even provide? Which use case is associated with it?